pkg/apis/rafter/v1beta1/asset_common_types.go

```diff
@@ -0,0 +1,159 @@
+package v1beta1
```

Comment:
Is it copy one to one from [rafter types](https://github.com/kyma-project/rafter/tree/master/pkg/apis/rafter/v1beta1)? Why these types are here? All `assets` types are not part of helm-broker.